homeassistant/components/bosch_shc/config_flow.py

```diff
@@ -191,9 +191,9 @@ async def async_step_zeroconf(
 
         try:
             hosts = (
-                discovery_info[zeroconf.ATTR_HOST]
-                if isinstance(discovery_info[zeroconf.ATTR_HOST], list)
-                else [discovery_info[zeroconf.ATTR_HOST]]
+                discovery_info.host
+                if isinstance(discovery_info.host, list)
+                else [discovery_info.host]
```

Comment:
@bdraco can the `host` in `zeroconf` be a list?
Is it a mistake in my zeroconf type hint or is this a mistake in this integration?

Reply:
`AsyncServiceInfo` returns a list, and we pick the address they hopefully want here: 
https://github.com/home-assistant/core/blob/dev/homeassistant/components/zeroconf/__init__.py#L528

Reply:
Thanks for the confirmation.
I have dropped multi-host handling from `async_step_zeroconf` and from the corresponding tests